ConsoleApp1/Board.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace ConsoleApp1
+{
+    class Board
+    {
+        private int _size;
+        private System.Timers.Timer _timer;
+        private System.Timers.Timer _spawnTimer;
+        private Snake _snake;
+        private Bonus _bonus;
+
+        public Board(int size = 15)
+        {
+            _size = size;
+            _snake = new Snake(size);
+            _timer = new System.Timers.Timer(1000);
+            _spawnTimer = new System.Timers.Timer(_size * 1100);
+            _timer.Elapsed += CheckEat;
```

Comment:
Better to create your custom event, OnBonusReached or OnFruitEaten, and create new bonus in the event handler

Reply:
added OnFruitEaten and Bonuseaten event